js/controllers/form/AjaxFormHandler.js

```diff
@@ -33,7 +33,6 @@
 		}
 
 		this.bind('refreshForm', this.refreshFormHandler_);
-		this.publishEvent('containerReloadRequested');
```

Comment:
Is there a risk here of unintended side effects? I admit that I'm often nervous about making changes like this because in my experience there would always be some corner of the UI  which I hadn't anticipated. I guess a search for `reloadContainer` below should identify any cases where it was used?

Reply:
I grepped and cleared these out; there were better solutions in all cases.